Jenkinsfile

```diff
@@ -43,7 +82,43 @@ stage("Build Docs") {
         git clean -fx
         python3 -m pip install git+https://github.com/zhanghang1989/d2l-book
         python3 -m pip install --force-reinstall ipython==7.16
-        python3 -m pip install --upgrade --force-reinstall -e .
+
+        pip uninstall -y autogluon
+        pip uninstall -y autogluon.vision
+        pip uninstall -y autogluon.text
+        pip uninstall -y autogluon.mxnet
+        pip uninstall -y autogluon.extra
+        pip uninstall -y autogluon.tabular
+        pip uninstall -y autogluon.core
+
+        cd core/
+        python3 -m pip install --upgrade -e .
+        cd ..
+
+        cd tabular/
+        python3 -m pip install --upgrade -e .
+        cd ..
+
+        cd mxnet/
+        python3 -m pip install --upgrade -e .
+        cd ..
+
+        cd extra/
+        python3 -m pip install --upgrade -e .
+        cd ..
+
+        cd text/
```

Comment:
Suppose a developer accidentally puts something in `text` that depends on dependency of `tabular` without adding dependency in `text`. How will this be detected, or is it something we just need to catch via code review?

Reply:
The build process builds packages in clean env in incremental order. If something is not in previous packages, the build will fail.